common/utils/src/test/java/org/apache/spark/util/SparkLoggerSuiteBase.java

```diff
@@ -140,15 +148,13 @@ public void testLoggerWithMDC() {
     Runnable errorFn = () -> logger().error(msgWithMDC, executorIDMDC);
     Runnable warnFn = () -> logger().warn(msgWithMDC, executorIDMDC);
     Runnable infoFn = () -> logger().info(msgWithMDC, executorIDMDC);
-    Runnable debugFn = () -> logger().debug(msgWithMDC, executorIDMDC);
```

Comment:
After https://github.com/apache/spark/pull/46405, the following 2 lines of code are no longer needed.